src/scripts/browserdeviceprofile.js

```diff
@@ -18,12 +18,27 @@ define(['browser'], function (browser) {
             }
         }
 
-        // Unfortunately haven't yet found a canPlayType for proper detection
-        if (browser.iOS && (browser.iOSVersion || 0) >= 11) {
-            return true;
+        if (browser.ps4) {
+            return false;
+        }
+
+        if (protocol === 'hls') {
+
+            //safari seems to be lying about this
```

Comment:
this is really strange, I would feel that the only browsers which in reality _can_ play hls with h265 are safari-based, have you tested this?

Reply:
H265 HLS doesn't seem to work on iOS13 but it did work on my iOS12 device. It's wired so I will remove that part now.